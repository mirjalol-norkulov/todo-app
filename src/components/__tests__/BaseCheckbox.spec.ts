import { expect, test } from "vitest";
import { mount } from "@vue/test-utils";
// import BaseCheckbox from "../BaseCheckbox.vue";

test("Renders correctly", () => {
  // expect(BaseCheckbox).toBeTruthy();
  // const wrapper = mount(BaseCheckbox, { props: { label: "Is completed" } });
  // expect(wrapper.text()).toContain("Is completed");
  // expect(wrapper.html()).toMatchSnapshot();
});
