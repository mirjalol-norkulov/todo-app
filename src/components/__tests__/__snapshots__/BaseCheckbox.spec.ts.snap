// Vitest Snapshot v1

exports[`Renders correctly 1`] = `"<button class=\\"px-4 py-2 rounded-sm transition-all duration-300 text-white transform active:scale-[0.95] px-4 py-2 bg-white text-black border border-gray hover:border-primary hover:text-primary\\">Hello</button>"`;
